mbed_flasher/erase.py

```diff
@@ -80,8 +82,23 @@ def reset_board(self, serial_port):
                 return EXIT_CODE_SERIAL_RESET_FAILED
         port.close()
         return EXIT_CODE_SUCCESS
+
+    def runner(self, drive):
+        start_time = time()
+        while True:
+            sleep(0.3)
+            if platform.system() == 'Windows':
+                out = os.popen('dir %s' % drive[0]).read()
```

Comment:
This needs to be converted to use `subprocess` module, because `os.popen` has been deprecated

If it raises an exception, the thread will terminate. I think it is good, if it visibly shows that exception, because it should not happen (unless some bad system misconfiguration?)